packages/auth/src/model/auth.ts

```diff
@@ -60,6 +61,7 @@ export interface ConfigInternal extends Config {
 export interface AuthInternal extends Auth {
   currentUser: User | null;
   emulatorConfig: EmulatorConfig | null;
+  recaptchaConfig: RecaptchaConfig | null;
```

Comment:
Since this isn't meant to be exposed to developers, please prefix it with `_` (this is the convention we follow for exported/shared fields that are not "public" to users of the library)

Reply:
Will this be exposed to developers even `AuthInternal` is marked as `@internal`?

Reply:
Not from a typescript point of view, but if you introspect the object in JS, the field will be visible. Since it's a public field, it won't be obfuscated by most minifiers.

Reply:
Got it, thanks!